defaults/main.yml

```diff
@@ -328,14 +328,15 @@ ubuntu1604cis_rsh_required: false
 ubuntu1604cis_ypbind_required: false
 
 # Time Synchronization
-ubuntu1604cis_time_synchronization: chrony
+# ubuntu1604cis_time_synchronization: chrony
```

Comment:
you'll want to set this to `systemd-timesyncd` instead of commenting it out. The role expects the variable to install whatever package is set for the value -- commenting it out will likely lead to an apply issue.